mmv1/third_party/tgc/tests/data/example_google_compute_autoscaler.tf

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+terraform {
+  required_providers {
+    google = {
+      source = "hashicorp/google-beta"
+      version = "~> {{.Provider.version}}"
+    }
+  }
+}
+
+provider "google" {
+  {{if .Provider.credentials }}credentials = "{{.Provider.credentials}}"{{end}}
```

Comment:
Looks like something about the credentials is failing in our CI test. Are you able to run this locally?

Reply:
Basically I ran gcloud auth application-default login and got the `path/application_default_credentials.json` file and replaced the `GOOGLE_OAUTH_ACCESS_TOKEN=fake-token` to `GOOGLE_APPLICATION_CREDENTIALS=path/application_default_credentials.json` to run locally this case.

Reply:
Hm, we're seeing the following error when running these tests:
            │ More details:
            │ Reason: authError, Message: Invalid Credentials

Reply:
I updated the .tf file with a little change. It should work now.